packages/flutter/lib/src/material/slider_theme.dart

```diff
@@ -1988,7 +2064,7 @@ class RoundSliderTickMarkShape extends SliderTickMarkShape {
 
   /// The preferred radius of the round tick mark.
   ///
-  /// If it is not provided, then half of the track height is used.
+  /// If it is not provided, then 1/4 of the track height is used.
```

Comment:
link to [SliderThemeData.trackHeight] here, instead of "track height"?

Here and elsewhere.